plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerPlugin.java

```diff
@@ -52,7 +52,7 @@
  *
  * @since 0.1.0
  */
-public final class LocationLayerPlugin implements LifecycleObserver {
+public final class LocationLayerPlugin implements LifecycleObserver, OnCameraMoveInvalidateListener {
```

Comment:
would be nicer to have an anonymous inner class so we don't add unnecessary public methods to the LocationLayerPlugin api

Reply:
`OnCameraMoveInvalidateListener` is package private, I don't see the issue here?

Reply:
@cammace When implementing an interface we are unnecessary exposing `LocationLayerPlugin#onInvalidateCameraMove`. Making it an anonymous inner class hides that method.